| Software                                                                             | Version                                                                                                      | Swset                                            | Architecture                                               | Category       | Clusters    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
|:-------------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------|:-------------------------------------------------|:-----------------------------------------------------------|:---------------|:------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <p><a href=http://bio-bwa.sourceforge.net/>BWA</a></p>                               | <p>0.7.17-GCC-8.3.0<br>0.7.17-GCC-8.3.0</p>                                                                  | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome.                                                                                                                                                                                                                                                                                                                                             |
| <p><a href=http://bowtie-bio.sourceforge.net/bowtie2/index.shtml>Bowtie2</a></p>     | <p>2.3.5.1-iccifort-2019.5.281<br>2.3.5.1-GCC-8.3.0<br>2.3.5.1-iccifort-2019.5.281<br>2.3.5.1-GCC-8.3.0</p>  | <p>2019b<br>2019b<br>2019b<br>2019b</p>          | <p>broadwell<br>broadwell<br>skylake<br>skylake</p>        | <p>Biology</p> | <p>iris</p> | Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes. |
| <p><a href=http://ccb.jhu.edu/software/tophat/>TopHat</a></p>                        | <p>2.1.2-gompi-2019b<br>2.1.2-iimpi-2019b<br>2.1.2-gompi-2019b<br>2.1.2-iimpi-2019b</p>                      | <p>2019b<br>2019b<br>2019b<br>2019b</p>          | <p>broadwell<br>broadwell<br>skylake<br>skylake</p>        | <p>Biology</p> | <p>iris</p> | TopHat is a fast splice junction mapper for RNA-Seq reads.                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| <p><a href=http://www.bioperl.org/>BioPerl</a></p>                                   | <p>1.7.2-GCCcore-8.3.0<br>1.7.2-GCCcore-8.3.0</p>                                                            | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects.                                                                                                                                                                                                                                                                                                                              |
| <p><a href=http://www.genome.umd.edu/jellyfish.html>Jellyfish</a></p>                | <p>2.3.0-GCC-8.3.0<br>2.3.0-GCC-8.3.0</p>                                                                    | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| <p><a href=http://www.gromacs.org>GROMACS</a></p>                                    | <p>2019.4-foss-2019b<br>2020-foss-2019b<br>2019.6-fosscuda-2019b<br>2019.4-foss-2019b<br>2020-foss-2019b</p> | <p>2019b<br>2019b<br>2019b<br>2019b<br>2019b</p> | <p>broadwell<br>broadwell<br>gpu<br>skylake<br>skylake</p> | <p>Biology</p> | <p>iris</p> | GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This is a CPU only build, containing both MPI and threadMPI builds.                                                                                                                                                                                                                                                                           |
| <p><a href=https://bedtools.readthedocs.io/>BEDTools</a></p>                         | <p>2.29.2-GCC-8.3.0<br>2.29.2-GCC-8.3.0</p>                                                                  | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | BEDTools: a powerful toolset for genome arithmetic. The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.                                                                                                                                                                                                                                |
| <p><a href=https://blast.ncbi.nlm.nih.gov/>BLAST+</a></p>                            | <p>2.9.0-gompi-2019b<br>2.9.0-gompi-2019b</p>                                                                | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.                                                                                                                                                                                                                                                                                                 |
| <p><a href=https://github.com/COMBINE-lab/salmon>Salmon</a></p>                      | <p>1.1.0-gompi-2019b<br>1.1.0-gompi-2019b</p>                                                                | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data.                                                                                                                                                                                                                                                                                                                                                                                      |
| <p><a href=https://github.com/pezmaster31/bamtools>BamTools</a></p>                  | <p>2.5.1-GCC-8.3.0<br>2.5.1-GCC-8.3.0</p>                                                                    | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.                                                                                                                                                                                                                                                                                                                                                                                                                     |
| <p><a href=https://pachterlab.github.io/kallisto/>kallisto</a></p>                   | <p>0.46.1-foss-2019b<br>0.46.1-foss-2019b</p>                                                                | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.                                                                                                                                                                                                                                                                                                                                                |
| <p><a href=https://trinityrnaseq.github.io>Trinity</a></p>                           | <p>2.10.0-foss-2019b-Python-3.7.4<br>2.10.0-foss-2019b-Python-3.7.4</p>                                      | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | Trinity represents a novel method for the efficient and robust de novo reconstruction of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm, Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.                                                                                                                                                                                                                             |
| <p><a href=https://www.bioinformatics.babraham.ac.uk/projects/fastqc/>FastQC</a></p> | <p>0.11.9-Java-11<br>0.11.9-Java-11</p>                                                                      | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline.                                                                                                                                                                                                    |
| <p><a href=https://www.htslib.org/>HTSlib</a></p>                                    | <p>1.10.2-GCC-8.3.0<br>1.10.2-GCC-8.3.0</p>                                                                  | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | A C library for reading/writing high-throughput sequencing data. This package includes the utilities bgzip and tabix                                                                                                                                                                                                                                                                                                                                                                                            |
| <p><a href=https://www.htslib.org/>SAMtools</a></p>                                  | <p>1.10-GCC-8.3.0<br>1.10-GCC-8.3.0</p>                                                                      | <p>2019b<br>2019b</p>                            | <p>broadwell<br>skylake</p>                                | <p>Biology</p> | <p>iris</p> | SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format.                                                                                                                                                                                                                                                                                                                                     |